user/tests/wiring/no_fixture/pwm.cpp

```diff
@@ -29,15 +29,26 @@
 
 uint8_t pwm_pins[] = {
 
-#if defined(STM32F2XX)
+#if (PLATFORM_ID == 0) // Core
```

Comment:
Yeah I don't know why, but we were not testing all PWM pins on all platforms before this change.  I was pleased to see it pass first time on all platforms though! :)